src/main/resources/Template.java

```diff
@@ -49,7 +49,7 @@ private static Web3j createWeb3jService(String url){
             if (System.getProperty("NODE_URL") == null || System.getProperty("NODE_URL").isEmpty()) {
                 if (System.getenv("NODE_URL") == null || System.getenv("NODE_URL").isEmpty()) {
                     log.info("Please make sure the node url is valid.");
-                    log.info("You can edit the node url programmatically, use java -DnodeURL=\"\" or as an environmental variable e.g export NODE_URL=\"\"");
+                    log.info("You can edit the node url programmatically, use java -DNODE_URL=\"\" or as an environmental variable e.g export NODE_URL=\"\"");
```

Comment:
Again I think we should have the NODE_URL value defined in a static final String. Does this change imply that v4.5.11 is showing the wrong -D output?

Reply:
On it.
Yes it shows the wrong output